staff/clara-rufi/Precurs2019/PasapalabraTimer/Pasapalabra.js

```diff
@@ -0,0 +1,413 @@
+        document.getElementById("userAnswer").value = ""
+        rep += 1
+
+    }else if (n <= 26){
+        n += 1
+        document.getElementById("userAnswer").value = ""
+        if(questions[n].status === 0){
+            document.getElementById("showQuestion").value = questions[n].question;   
+            
+            }else if (questions[n].status !== 0){
+                for (var j = n; j<questions.length; j++){
+                    if(questions[j].status === 0){
+                        n = j 
+                        document.getElementById("showQuestion").value = questions[j].question;
+                        break;
+                    }    
+                    }       
+            }      
+
+
+    }
+}
+
+function showresults(){
+    document.getElementById('letters').style="display:none;"
+    document.getElementById('showQuestion').style="display:none;"
+    document.getElementById('userAnswer').style="display:none;"
+    document.getElementById('showresults').style="display:none;"
+    document.getElementById("end").style="display:none;"
+    document.getElementById('results').style="display:block;"
+
+
+    var participantes = [
+
+        {nombre: "Marta" , puntuacion: 26},
+        {nombre: "Jofre" ,puntuacion: 15},
+        {nombre: "Gal.la" , puntuacion: 10},
+        {nombre: "Jaume" , puntuacion: 2},    
+    ]
+    
+    participantes.nombre = nameUser
+    participantes.puntuacion = numAcertadas
+
+    participantes.push({nombre: nameUser, puntuacion: numAcertadas})
+    
+    participantes.sort(function (a, b) {
+        if (a.puntuacion > b.puntuacion) {
+          return -1;
+        }
+        if (a.puntuacion < b.puntuacion) {
+          return 1;
+        }
+        return 0;
+      });
+
+ranking = ""
+
+for(var i=0; i<participantes.length ; i++){
+    ranking += "- " + participantes[i].nombre + " => " + participantes[i].puntuacion + "<p>"
+}
+
+document.getElementById("rankingJugadores").innerHTML =  "<p>" + "<li>"  + ranking + "</li>" 
+document.getElementById('playAgainYes').style="display:block;"
+document.getElementById('playAgainNo').style="display:block;"
+document.getElementById('questionPlayAgain').style="display:block;" 
+
+}
+
+function playAgainYes(){
+    document.getElementById("questionPlayAgain").style="display:none;"
+    document.getElementById("playAgainYes").style="display:none;"
+    document.getElementById("playAgainNo").style="display:none;"
+    document.getElementById("results").style="display:none;"
+    document.getElementById("timeFinished").style="display:none;"
+    document.getElementById("time").style="display:block;"
+    document.getElementById("tiempo").style="display:block;"
+    document.getElementById("minuts").style="display:block;"
+    /*document.getElementById("userAnswer").value = "";
+    document.getElementById("showQuestion").value = "";*/
+
+
+    timer();
+    
+    
+    window.onload = function () {
```

Comment:
This `onload` event is not necessary. You are calling the `startTimer` function before.